llvm/test/CodeGen/PowerPC/peephole-replaceInstr-after-eliminate-extsw.mir

```diff
@@ -0,0 +1,702 @@
+# RUN: llc -run-pass=ppc-mi-peepholes  -mtriple powerpc64-ibm-aix-xcoff %s -o - \
```

Comment:
Could we commit this test case first before this patch? Then, it will be easier to see how this patch changes the test case.